developer-docs/plugin-guide.md

```diff
@@ -35,84 +36,37 @@ Pupil Player uses an identical plugin structure. Little (often no work) needs to
 
 ### Plugin API
 
-Plugins inherit the [`Plugin` class](https://github.com/pupil-labs/pupil/blob/v0.9.12/pupil_src/shared_modules/plugin.py#L23). It provides default functionality as well as series of callback functions that are called by the world process. [The source](https://github.com/pupil-labs/pupil/blob/v0.9.12/pupil_src/shared_modules/plugin.py#L23) contains detailed information about the use-cases of the different callback functions.
+Plugins are Python classes that inherit the [`Plugin` class](https://github.com/pupil-labs/pupil/blob/568604bd06a972eeac533f74079c22459b6e929e/pupil_src/shared_modules/plugin.py#L23). It provides default functionality as well as series of callback functions that are called by the world process. [The source](https://github.com/pupil-labs/pupil/blob/568604bd06a972eeac533f74079c22459b6e929e/pupil_src/shared_modules/plugin.py#L23) contains detailed information about the use-cases of the different callback functions.
```

Comment:
This should always link to the master branch. The link is: https://github.com/pupil-labs/pupil/blob/master/pupil_src/shared_modules/plugin.py

Reply:
Or are you linking it to a specific commit (e.g. a v1.0 commit ?) I think it woudl be cleaner just to link this to master for future reference.